src/Events/Authorization/SubscriptionXacmlMapper.cs

```diff
@@ -66,13 +59,30 @@ private static XacmlJsonCategory CreateActionCategory(string actionType, bool in
             return actionAttributes;
         }
 
-        private static XacmlJsonCategory CreateEventsResourceCategory(string org, string app, string subscriptionSubjectFilter)
+        private static XacmlJsonCategory CreateResourceCategory(Subscription subscription, bool isAppEventSubs)
         {
             XacmlJsonCategory resourceCategory = new()
             {
                 Attribute = new List<XacmlJsonAttribute>()
             };
 
+            (string resourceFilterId, string resourceFilterValue) = XacmlMapperHelper.SplitResourceInTwoParts(subscription.ResourceFilter);
+
+            resourceCategory.Attribute.Add(DecisionHelper.CreateXacmlJsonAttribute(resourceFilterId, resourceFilterValue, DefaultType, DefaultIssuer));
+
+            if (isAppEventSubs)
+            {
+                string[] resourceAttParts = resourceFilterValue.Split('.');
+                string org = resourceAttParts[1];
+                string app = resourceAttParts[2];
+                AddAppResourceAttributes(resourceCategory, subscription.SubjectFilter, org, app);
```

Comment:
I need explained why a "resource attribute" is added in App context, but not for generic events.

Reply:
Line 71 adds the resource attribute for all subscriptions regardless of app and generic.What is done within this if check is to inclued the org, app attributes for an app subscription as well as the subject filter.